qa/tasks/mgr/dashboard/test_auth.py

```diff
@@ -8,6 +8,7 @@
 
 from .helper import DashboardTestCase, JObj, JLeaf
 
+DEFAULT_VERSION = '1.0'
```

Comment:
I think we should put this literal in a single file and simply import that... Not a fan of the same literal defined in multiple places.